src/components/MobileNavbar/MobileNavbar.scss

```diff
@@ -0,0 +1,88 @@
+@import "../../assets/styles/theme";
+
+
+.MobileNavbar {
+    z-index: 10;
+    background: $white;
+    color: black;
+    display: flex;
+    align-items: center;
+    align-self: center;
+    top: 0;
+    height: 108px;
+    width: 100%;
+    white-space: nowrap;
+    position: -webkit-sticky;
+    position: sticky;
+    overflow: hidden;
+    font-family:  Helvetica;
+    box-sizing: border-box;
+    -webkit-box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+    -moz-box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+
+}
+.LogoContainer {
+    flex-grow: 2;
+    display: flex;
+    justify-content: baseline;
+    margin-bottom: 0.5rem;
+    color: $white;
+    
+}
+
+.Logo {
+    margin-left: 3.5rem;
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 1.5rem;
+
+}
+.Logo img{
+    
+    height: 26px;
+    width: 19px;
+}
+
+.Name {
+    margin: 0.3rem;
+   
+}
+
+.Name img {
+    height: 15px;
+    width: 68px;
+}
+
+
+.Language {
+    flex-grow: 2;
+    margin-left: 10%;
+    display: flex;
+    align-items: baseline;
+    font-size: 19px;
+}
+
+.Language #ENG {
+    color: rgba(103, 103, 103, 0.668);
```

Comment:
Mamy zmienną od tego koloru: `$grey_30`

TIP: Gdy zaznaczymy sobie konkretny element w figmie, to po prawej jest napisana nazwa zmiennej koloru, zmienne w figmie pokrywają się ze zmiennymi w kodzie.